src/clj/ataru/applications/excel_export.clj

```diff
@@ -289,6 +298,7 @@
         get-form-by-id          (memoize form-store/fetch-by-id)
         get-latest-form-by-key  (memoize form-store/fetch-by-key)]
     (->> applications
+         (map update-hakukohteet)
```

Comment:
Maybe rename this so that it's clearly only for legacy old fashioned applications?